src/SingleProductDetailView.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import { Image } from "./Image.js";
+import "./SingleProductDetailView.css";
+
+export class SingleProductDetailView extends Component {
+  state = { product: {} };
+
+  componentDidMount() {
+    fetch("http://localhost:8000/listings/638754067")
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ product: data.results[0] });
+      })
+      .catch(error => console.error(error));
+  }
+
+  render() {
+    const { product } = this.state;
+
+    if (product.listing_id == null) {
+      return null;
+    }
+    function renderDescription() {
+      const description = product.description;
+      const newDescription = description.slice(0, 131);
+      return newDescription;
+    }
+
+    function renderTitle() {
+      const title = product.title;
+      const newTitle = title.slice(0, 11);
+      return newTitle;
+    }
+
+    function renderPrice() {
+      const price = product.price;
+
+      return `$${price}`;
+    }
+
+    return (
+      <div className="single-product-detail-view-wrapper">
```

Comment:
do you need the `wrapper` and the `container` to be separate? they're both wrapping all the other content, so could they just be one `<div>`?